Sources/CryptoSwift/LinuxFoundation.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  LinuxFoundation.swift
+//  CryptoSwift
+//
+//  Created by Alsey Coleman Miller on 4/18/16.
+//  Copyright © 2016 Marcin Krzyzanowski. All rights reserved.
+//
+
+import Foundation
+
+#if !os(Linux)
+    
+extension String {
+    
+    // Workaround:
+    // https://github.com/krzyzanowskim/CryptoSwift/issues/177
+    @inline(__always)
+    func bridge() -> NSString {
+        return self as NSString
```

Comment:
this workaround was added for Linux only. Now if it's not needed (I assume it's not because this code is for OSX only) no need to have it any longer.